doc-surrealdb_versioned_docs/version-1.x/tutorials/define-a-schema.mdx

```diff
@@ -0,0 +1,230 @@
+
+```surql
+-- Using the CREATE statement 
+CREATE IC_directory:{username: 'johndoe',full_name: 'John Doe'} CONTENT {
+	username: 'johndoe',
+	full_name: 'John Doe',
+	email: 'johndoe@example.com',
+	date_of_birth: "1990-01-01",
+	join_date: "2024-05-30",
+	department: 'Engineering',
+	role: 'Software Engineer',
+	skills: ['Python', 'JavaScript', 'surql'],
+	manager: manager_directory:janesmith,
+	tags: ['full-time', 'remote']
+};
+
+-- Using the UPDATE statement 
+UPDATE manager_directory:janesmith CONTENT {
+	username: 'janesmith',
+	full_name: 'Jane Smith',
+	email: 'janesmith@example.com',
+	date_of_birth: "1985-01-01",
+	join_date: "2019-05-30",
+	department: 'Engineering',
+    role: 'Software Engineer Manager',
+	skills: ['Python', 'JavaScript', 'surql'],
+	report: IC_directory:{ full_name: 'John Doe', username: 'johndoe' },
+	tags: ['full-time', 'remote']
+};
+```
+
+In the example above, you used the `CREATE` statement to make an `IC_directory` table and created an Object ID for this table. ID can be anything you want and is randomly generated if not defined. You have also used the UPDATE statement to make a `manager_directory`, which has `janesmith` as the ID. Learn more about [Record IDs in the documentation](/docs/surrealdb/surrealql/datamodel/ids#object-based-record-ids).
+
+Notice how we have linked these two tables with the manager and report fields in the `IC_directory` and `manager_directory` tables, respectively.
+
+You can also start defining a Schemaless table using the `DEFINE TABLE` statement.
+
+```surql
+-- Create schemaless user table.
+DEFINE TABLE user SCHEMALESS;
+
+-- Define some fields.
+DEFINE FIELD firstName ON TABLE user TYPE string;
+DEFINE FIELD lastName ON TABLE user TYPE string;
+DEFINE FIELD email ON TABLE user TYPE string;
+```
+
+In the example above you have created a schemaless table using the `SCHEMALESS` clause. 
+
+### Inserting fields that don’t exist in the schema
+
+Since a schemaless table doesn’t have any restrictions on the structure if you introduce a new field to an existing table the column will be added to the table even if other records don’t have the value. For example, take the schemafull example of introducing  a `photoURI` field without defining the field in the `user` table:
+
+```surql
+CREATE user CONTENT {
+    firstName: 'John',
+    lastName: 'Doe',
+    email: 'JohnDoe@someemail.com',
+    photoURI: 'photo/yxCFi22Jw2.webp'
+};
+```
+
+Since the `user` table is schemaless the `photoURI` field will be added for `John` without an error. 
+
+## Combining Schemafull and Schemaless
+
+Now that you have seen how you can make a table in SurrealDB deciding what to go with depends on the restrictions you want to have on your Schema. 
+
+Since SurrealDB is multi-model, If some tables require less restrictions you can make them schemaless and if you want to ensure that the structure remains the same for each entry then go with the schemafull option. 
```

Comment:
As a multi-model database, SurrealDB allows tables requiring fewer restrictions to be schemaless, while also offering the schemafull option if you want to ensure that the structure remains the same for each entry.